app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'AuthorId'
+  belongs_to :post, class_name: 'Post', foreign_key: 'PostId'
```

Comment:
- [ ] [OPTIONAL] The beauty of Rails is that it knows by default which foreign key names to give to the attributes being mapped to. For this line of code, when we say `belongs_to :post`, Rails will know that a comments table should have an attribute called `post_id`. We don't even need `class_name: 'Post', foreign_key: 'postId'`